_static/api-examples.md

```diff
@@ -152,7 +152,7 @@ You can sign a manifest with either the e-Manifest ID:
 ```bash
 curl -i -X POST -H 'Content-Type: application/json' \
   --data @sign-manifest-emanifestid.json \
-  https://e-manifest.18f.gov/api/0.1/manifest/sign
+  https://e-manifest.18f.gov/api/v1/manifests/:manifest_id/sign
```

Comment:
If we follow Postel's law (https://en.wikipedia.org/wiki/Robustness_principle) then I would probably make the API look like:

```
POST .../manifest/:id_or_tracking_number
```

and then have the code dis-ambiguate it. That's one reason to use a UUID for the e-Manifest ID, so that it is easier to disambiguate. If we made the UUID follow a template like `em<random30charstring>` then it would be straightforward because afaik, manifest tracking numbers are always numeric.